### PR TITLE
Add Digital Matter Oyster and Yabby Edge

### PR DESCRIPTION
**Pull request by jamesl-dm**
#### Summary
Added two devices and a company logo

#### Changes
- Added Digital Matter logo and website
- Added Oyster and Yabby Edge devices

### CODE REVIEW

File: vendor/digital-matter/oyster-codec.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+uplinkDecoder:
+  fileName: oyster.js
+  examples:
+    - description: Position update
+      input:
+        fPort: 1
+        bytes: [0x00, 0x30, 0xed, 0xec, 0x00, 0x32, 0x24, 0x45, 0x00, 0x00, 0xde]
+      output:
+        data:
+          batV: 5.550000000000001
```
Comment (johanstokking) [inline vendor/digital-matter/oyster-codec.yaml:10]: Ouch. Can you do some rounding in your decoder?

**Review by johanstokking — APPROVED** [review]
Thanks, just one question!

File: vendor/digital-matter/oyster-codec.yaml (hunk 1 above)
Comment (jamesl-dm) [reply vendor/digital-matter/oyster-codec.yaml:10]: It is awful, but I'm not really sure how...
I've tried several approaches, but nothing has worked so far.
If you know a way that works, and is supported on TTNs javascript VM, I'd love to use it.

Comment (johanstokking) [reply vendor/digital-matter/oyster-codec.yaml]: Here you go: https://github.com/TheThingsNetwork/lorawan-devices/pull/52/commits/e322e6b44b72ac5365f8f25c35627bd87b2c142e

Comment (jamesl-dm) [reply vendor/digital-matter/oyster-codec.yaml]: Thanks! I was just working something up, but yours is better :)

**Pull request merged**